tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2385,6 +2385,36 @@ Status ConvertExpandDims(OpConverterParams* params) {
   return Status::OK();
 }
 
+Status Converter::SqueezeTensor(nvinfer1::ITensor* input,
+                                const std::vector<int>& trt_axes,
+                                nvinfer1::ITensor** output) {
+  const nvinfer1::Dims dims = input->getDimensions();
+  std::vector<int> input_dims(dims.d, dims.d + dims.nbDims);
+  // Mark axes to remove by setting them to 0.
+  for (int axis : trt_axes) {
+    input_dims[axis] = 0;
+  }
+
+#if IS_TRT_VERSION_GE(6, 0, 0, 0)
+  // For dynamic input shapes, we need to use TRT ops to build the new shape.
+  const bool is_dynamic =
+      std::count(input_dims.begin(), input_dims.end(), -1) > 0;
+  if (is_dynamic) {
```

Comment:
You may want to simply use if (absl::c_any_of(input_dims, [](int) {return i == -1;})) {